packages/test-utils/src/reporter.ts

```diff
@@ -250,10 +253,16 @@ class ReplayReporter<TRecordingMetadata extends UnstructuredMetadata = Unstructu
   ) {
     this._runner = runner;
     this._schemaVersion = schemaVersion;
+    this._logger = new Logger(this._runner.name);
+
     if (config) {
       const { metadataKey, ...rest } = config;
       this._parseConfig(rest, metadataKey);
     }
+
+    if (this._apiKey) {
+      this._cacheAuthIdsPromise = getAuthIds(this._apiKey).then(ids => this._logger.identify(ids));
```

Comment:
The `.catch` is only attached in `onEnd`. If it gets rejected before that is attached then this rejection might take down the node process

Reply:
Ah yeah, my mistake.